spec/koans.js

```diff
@@ -42,39 +42,39 @@ describe('`const` is like `let` plus read-only. ', () => {
 
   describe('scalar values are read-only', () => {
     it('number are read-only', () => {
-      // const constNum = 0;
-      // constNum = 1;
+      const constNum = 0;
+      // constNum = 1;    NOOOOOOO !
 
-      //expect(constNum).toBe(0);
+      expect(constNum).toBe(0);
     });
 
     it('string are read-only', () => {
-      // const constString = "I am a const";
+     const constString = "I am a const";
       // constString = "Cant change you?";
 
-      //expect(constString).toBe("I am a const");
+      expect(constString).toBe("I am a const");
     });
 
   });
 
-  /*var, let or const? notChangeable = 23;*/
+  var notChangeable = 23;
```

Comment:
this should stay a const, the point of this if to show that if we check its value inside of an if statement, we can still access it, only if you declare a const inside an if statement or a for loop than you cannot access it from outside this block